### PR TITLE
Bands index

### PR DESCRIPTION
**Pull request by phillippeet**

### CODE REVIEW

File: app/controllers/bands_controller.rb [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+  end
+
+  def new
+    @band = Band.new
+  end
+
+  def create
+    @band = Band.new(band_params)
+    # TODO Replace User.first with current_user when available
+    @band.user = User.first
```
Comment (Raajayi) [inline app/controllers/bands_controller.rb]:
```suggestion
    # TODO Replace User.first with current_user when available
    @band.user = current_user
```
current_user is available

**Review by Raajayi — CHANGES_REQUESTED** [review]
1 small change to make. Should be able to just accept the suggested code change

**Comment by phillippeet** [comment]
@Raajayi completed change :)

**Review by Raajayi — APPROVED** [review]
LGTM

**Pull request merged**